### DISCUSSION TITLE
State of the Project

### DISCUSSION THREAD

**Discussion by einekratzekatze** [xerpi/vita2hos#29] [Q&A] [answered]
Is vita2hos still actively developed, and can we expect bigger titles to work any time soon?

**Comment by TSRBerry** [comment, marked as answer]
@xerpi recently said something about this on Discord:

> I'm currently waiting for two things in order to advance with the project:
> 1 - fincs' (deko3d developer) feedback about HW and deko3d support on rendering to color and depth/stencil with different tilings (for example linear color fb,  tiled d/s surface)
> 2 -Merge of Macdu's Vita3k Vulkan backend

So yes, vita2hos is still actively developed, although xerpi has been waiting for feedback for quite a while now. I don't think you expect to play any bigger titles to start working any time soon, so just know this project has not been abandoned and is just facing a few roadblocks at the moment which makes things pretty difficult.
